### ISSUE TITLE
Configuration data not installed in venv for python 3.10

### ISSUE THREAD

**Issue by magnusuMET** [metno/pyaerocom#616]
Steps to replicate (on ppi-r8login-b)
```sh
module use /modules/MET/rhel8/user-modules/fou-modules
module load gcc geos proj udunits python
python3.10 -m venv TESTENV
source TESTENV/bin/activate
pip install .
```
The resulting venv has an empty (only \_\_init__.py) directory for `pyaerocom.data`, whereas e.g. python3.8 has configuration files in this directory.

Assign: @avaldebe

**Comment by magnusuMET** [comment]
This causes errors on import with `KeyError: 'formatters'` since `logging.ini` is not found (as it does not exist in the installation)

**Comment by magnusuMET** [comment]
Globs seems to be more specific, adding this to MANIFEST.IN seems to work
```txt
include pyaerocom/data/*.ini
```

**Comment by avaldebe** [comment]
This was solved on 779303f, please update your branch by merging `main-dev` into it or cherry-pick the commit.

Also, is this a "production" environment? or a development environment?

> ```shell
> module use /modules/MET/rhel8/user-modules/fou-modules
> module load gcc geos proj udunits python
> python3.10 -m venv TESTENV
> source TESTENV/bin/activate
> pip install .
> ```

You should not install a working copy into a "production" env.

``` bash
python3.10 -m venv TESTENV
source TESTENV/bin/activate
pio install -U pip
pip install git+http:https://github.com/metno/pyaerocom.git@branch-or-tag-or-sha
```
